### PR TITLE
[FRON-1542]: Console release

### PR DESCRIPTION
**Pull request by pleeko**
Updates to the release process. I have tested these changes by doing a release off of this branch [here](https://app.circleci.com/pipelines/github/pachyderm/pachyderm/28967/workflows/e066f669-edc0-4fc1-8429-dcd8ebd51010).

### CODE REVIEW

File: etc/build/push_docker_with_manifests.sh [hunk 1]
```diff
@@ -27,3 +27,5 @@ for product in pachd worker pachctl; do
     docker manifest inspect "$REPO/$product:$VERSION"
     docker manifest push "$REPO/$product:$VERSION"
 done
+
+# TODO: add console
```
Comment (pleeko) [inline etc/build/push_docker_with_manifests.sh:31]: I don't think we actually want to add console here 🤔

File: .circleci/main.yml [hunk 2]
```diff
@@ -2164,6 +2095,47 @@ workflows:
             - python-sdk-test-publish
             - sign-off
           filters: *only-release-tags
+      - console-lint-backend:
+          filters: *only-release-tags      
+      - console-lint-frontend:
+          filters: *only-release-tags    
+      - console-lint-cypress:
+          filters: *only-release-tags   
+      - console-build:
```
Comment (pleeko) [inline .circleci/main.yml:2104]: I don't think we need to run these for the release. Build is handled by the docker build. I was just mirroring what we have in the console workflow.

**Review by molinamelendezj — APPROVED** [review]
i like all this. but @pleeko i think we need to remove that `v` in front of the version that will break the helm install

File: console/Makefile [hunk 3]
```diff
@@ -13,12 +13,6 @@ ifneq ($(filter aarch%,$(UNAME_P)),)
 	TARGET_ARCH = arm64
 endif
 
-ifdef CIRCLE_TAG
```
Comment (jeffrifwald) [inline console/Makefile:16]: Have we confirmed that removing this is still sufficient for both tagged releases and simple PR SHA tags?

Comment (pleeko) [reply console/Makefile:16]: Yep. What we are doing now is building the image with the sha and once all the release steps pass and we get sign off, the `console-docker-push-tag` job gets triggered. This takes the images that were created and pushed with the SHA and creates a new manifest using the provided circle tag.

**Comment by pleeko** [comment]
you can test this release out with `helm install pachd pach/pachyderm --version=0.0.0-test.4 --set deployTarget=LOCAL`

File: .circleci/main.yml [hunk 4]
```diff
@@ -1197,61 +1197,6 @@ jobs:
                 echo "regular release."
                 gh release create --draft ${CIRCLE_TAG} --title ${CIRCLE_TAG:1} ./cr-release-packages/local_template/*.yaml ./dist-pach/pachctl/pachctl_${CIRCLE_TAG:1}* ./dist-pach/docker/pachyderm_*_linux_amd64.tar.gz ./dist-pach/docker/pachyderm_*linux_arm64.tar.gz --generate-notes
             fi
-  console-release-draft:
```
Comment (pierrelamarche) [inline .circleci/main.yml:1200]: 👋

**Review by pierrelamarche — APPROVED** [review]

**Pull request merged**